tests/test_encoder.py

```diff
@@ -213,3 +213,20 @@ def test_encode_property_list(self):
             expected_geometry=[[0, 0]],
             properties=properties,
             expected_properties={'test_empty': ''})
+
+    def test_encode_multiple_values(self):
+        geometry = 'POINT(0 0)'
+        properties = dict(foo='bar')
```

Comment:
I tried a naive test yesterday with the exact same properties, and it didn't catch the error. I originally thought that it meant that the behavior was the same, just that it hard more entries in the tags table. But, it turns out that the results _are_ different if the properties are varied. I've updated the test to also catch this case using a round trip in 063e84e.

Thanks for noticing this and questioning it!